sherpa-onnx/csrc/context-graph.cc

```diff
@@ -10,6 +10,7 @@
 
 namespace sherpa_onnx {
 void ContextGraph::Build(const std::vector<std::vector<int32_t>> &token_ids) {
+  assert(false == is_populated_);
```

Comment:
I suggest that you use `SHERPA_ONNX_CHECK()` or `SHERPA_ONNX_DCHECK()` to replace `assert()`.
It will give you a more informative message on assertion failure.